<launch>
    <node pkg="odml_visual_odometry" type="visual_odometry_node" name="visual_odometry_node" output="screen">
        <param name="is_classic" type="bool" value="true" />
        <param name="detector_type" type="string" value="ORB" />
        <param name="descriptor_type" type="string" value="ORB" />
        <param name="matcher_type" type="string" value="BF" />
        <param name="selector_type" type="string" value="KNN" />
        <param name="stereo_threshold" type="double" value="2.0" />
        <param name="min_disparity" type="double" value="2.0" />
        <param name="refinement_degree" type="int" value="4" />
        <param name="verbose" type="bool" value="false" />
    </node>
</launch>